### PR TITLE
[JENKINS-64289] Prevent weird character in noJob home page

### PR DESCRIPTION
**Pull request by Wadeck**
<details>

<summary>Screenshots</summary>

### Before
Screenshot_2020-11-23_131527_001

### After
Screenshot_2020-11-23_131806_001

</details>

It seemed to be correct on the first screenshot in https://github.com/jenkinsci/jenkins/pull/4926

See [JENKINS-64289](https://issues.jenkins-ci.org/browse/JENKINS-64289).

### Proposed changelog entries

* No changelog

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [n/a] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@fbelzunc 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by fqueiruga — APPROVED** [review]
LGTM. Can you please create a ticket? I think this is a regression / lts-candidate

File: core/src/main/resources/hudson/model/AllView/noJob.groovy [hunk 1]
```diff
@@ -111,7 +111,7 @@ div {
 
             h1(_("Welcome to Jenkins!"))
 
-            p("Log in now to view or create jobs. If you don’t already have an account, you can sign up.")
+            p("Log in now to view or create jobs. If you don't already have an account, you can sign up.")
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/AllView/noJob.groovy:114]:
```suggestion
            p(_("blurb"))
```

Should be made localizable anyway.

Comment (Wadeck) [reply core/src/main/resources/hudson/model/AllView/noJob.groovy:114]: Out of scope :)

**Review by timja — APPROVED** [review]

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Please follow the pull request template @Wadeck , especially since it is an LTS candidate.
The change itself looks good to me

**Review by res0nance — APPROVED** [review]

**Comment by Wadeck** [comment]
@oleg-nenashev Template used, TBH I didn't expect such minor change to be backported :)

**Comment by oleg-nenashev** [comment]
There is a label in Jira,  so I acted based on that. No problem with backporting minor issues when they are safe

**Comment by Wadeck** [comment]
Anyone was able to reproduce the case?

I got exact same issue with `Jenkins’ own user database` in Security page, when running with `jetty:run` but *not* when running with `docker run jenkins/jenkins:2.267`. 

So perhaps a recent regression in jetty? IOW at this point I am not sure it's impacting anyone outside of Jenkins developers.

File: core/src/main/resources/hudson/model/AllView/noJob.groovy (hunk 1 above)
Comment (daniel-beck) [reply core/src/main/resources/hudson/model/AllView/noJob.groovy:114]: Touching a line and not fixing the obvious bug on it is 👎

**Review by daniel-beck — CHANGES_REQUESTED** [review]
https://github.com/jenkinsci/jenkins/pull/5080 is far better.

**Comment by jglick** [comment]
Since #5080 was merged, should this be closed?

**Pull request closed without merging**